build.gradle

```diff
@@ -85,6 +85,7 @@ dependencies {
     implementation "com.amazonaws:aws-java-sdk-sns:1.12.349"
     implementation "org.apache.kafka:kafka-clients:3.3.1"
     implementation "com.google.firebase:firebase-admin:9.1.1"
+    implementation 'org.eclipse.paho:org.eclipse.paho.client.mqttv3:1.2.5'
```

Comment:
First of all, looks like this library is basically abandoned. Secondly, I think it's better to use existing Netty library to communicate via MQTT.